girder/models/token.py

```diff
@@ -131,6 +132,21 @@ def hasScope(self, token, scope):
             scope = (scope,)
         return set(scope).issubset(set(self.getAllowedScopes(token)))
 
+    def requireScope(self, token, scope):
+        """
+        Raise an error if given set of scopes are not included.
+
+        :param token: The token object.
+        :type token: dict
+        :param scope: A scope or set of scopes that will be tested as a subset
+            of the given token's allowed scopes.
+        :type scope: str or list of str
+        """
+
+        if not self.hasScope(token, scope):
+            raise AccessException('Invalid token scope.'
+                                  'Required: %s.' % (scope))
```

Comment:
I think we can fit this all on one line, but if not, we'll need to add a space between the sentences.